integration/tests/economy/test_economics.py

```diff
@@ -785,6 +795,26 @@ def test_call_contract_with_mapping_updating(
         assert receipt["status"] == 1
         wait_condition(lambda: sol_balance_before != operator.get_solana_balance())
 
+        solana_trx = web3_client.get_solana_trx_by_neon(receipt["transactionHash"].hex())
+        sol_trx_with_alt = None
+
+        for trx in solana_trx["result"]:
```

Comment:
Looks like this part is copied from another test. Maybe it's better to move common part to some function?

Reply:
Moved to separate function